packages/build-utils/test/unit.get-project-paths.test.ts

```diff
@@ -0,0 +1,42 @@
+import path from 'path';
+import { normalizePath } from '../src';
+import { getProjectPaths, ProjectPath } from '../src/get-project-paths';
+import { FixtureFilesystem } from './utils/fixture-filesystem';
+
+describe.each<[ProjectPath[], number, number, string, ProjectPath[]?]>([
```

Comment:
Perhaps this should use the object notation instead of array so that each parameter can be named